website/content/contribute/code/flow/index.md

```diff
@@ -23,7 +23,7 @@ Here’s how to submit a pull request:
 - **Clone the forked repository locally**. Connect your local to the original “upstream” repository by adding it as a remote.
     ```shell
     git clone git@github.com:$(git config user.name)/{{< github_repo >}}.git
-    git remote add upstream git@github.com:IBM/{{< github_repo >}}.git
+    git remote add upstream https://github.com/IBM/the-mesh-for-data.git
```

Comment:
please use supported link format:
use: https://{{< github_base >}}/{{< github_repo >}/{{< github_repo >}}.git  instead of hardcoded links.

Reply:
Fixed